src/client/cmd/launch.cpp

```diff
@@ -258,14 +258,12 @@ mp::ReturnCode cmd::Launch::request_launch()
             if (error == LaunchError::INVALID_DISK_SIZE)
             {
                 error_details =
-                    fmt::format("Invalid disk size value supplied: {}. Note that a minimum of {} is required.",
-                                request.disk_space(), min_disk_size);
+                    fmt::format("Invalid disk size value supplied: {}.", request.disk_space(), min_disk_size);
```

Comment:
This doesn't mention the minimum disk size now, but still passes it to `fmt::format()`?

Reply:
Oops, nice catch. I removed the "minimum size" hint because, when testing, I found it was actually confusing when the cause for failure was something else. But did forget that, will fix. (When we have time to spare, we should move to format's compilation time checking)